trl/models/utils.py

```diff
@@ -0,0 +1,79 @@
+from dataclasses import dataclass
+from typing import Literal, Tuple
+
+from transformers import PreTrainedModel, PreTrainedTokenizer
+
+
+@dataclass
+class ChatMlSpecialTokens:
```

Comment:
maybe create an abstract class with abstract methods so that future data classes can use the same template?

Reply:
should we add this now or when we have more templates? I would rather add if there is more request and add a note?